extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -281,9 +281,22 @@ define('ext.wikia.adEngine.adContext', [
 		callbacks.push(callback);
 	}
 
+	function get(path) {
+		var isInputVariableCorrect = path === undefined || path === '',
```

Comment:
Nit-picking: how about changing `isInputVariableCorrect` to `isPathValid`? ;)

Reply:
...oooopsss, shame :bell: